tests/chainerx_tests/unit_tests/routines_tests/test_math.py

```diff
@@ -539,6 +539,25 @@ def test_log_softmax_invalid(device, a_shape, axis, float_dtype):
     with pytest.raises(chainerx.DimensionError):
         return chainerx.log_softmax(a, axis=axis)
 
+@op_utils.op_test(['native:0', 'cuda:0'])
```

Comment:
Can you add an additional blank line on top of this line? The CI style check will complain on this.